bigchaindb/tendermint/core.py

```diff
@@ -140,6 +141,10 @@ def commit(self):
             # this effects crash recovery. Refer BEP#8 for details
             self.bigchaindb.store_block(block._asdict())
 
+        logger.debug('Commit-ing new block with hash: ',
```

Comment:
You can use "committing". Please use the `%s` operator for logging.

`logger.debug('one=%s, two=%s', a, b)`